docs/source/en/modular_diffusers/getting_started.md

```diff
@@ -0,0 +1,1620 @@
+  "_diffusers_version": "0.35.0.dev0",
+  "guidance_rescale": 0.0,
+  "guidance_scale": 7.5,
+  "start": 0.0,
+  "stop": 1.0,
+  "use_original_formulation": false
+}
+```
+
+#### Modify Parameters of the Same Guider Type
+
+To change parameters of the same guider type (e.g., adjusting the `guidance_scale` for CFG), you have two options:
+
+**Option 1: Use ComponentSpec.create() method**
+```python
+>>> guider_spec = t2i_pipeline.get_component_spec("guider")
+>>> guider = guider_spec.create(guidance_scale=10)
+>>> t2i_pipeline.update_components(guider=guider)
+```
+
+**Option 2: Pass ComponentSpec directly**
+```python
+>>> guider_spec = t2i_pipeline.get_component_spec("guider")
+>>> guider_spec.config["guidance_scale"] = 10
+>>> t2i_pipeline.update_components(guider=guider_spec)
+```
+
+Both approaches produce the same result:
+```python
+>>> t2i_pipeline.guider
+ClassifierFreeGuidance {
+  "_class_name": "ClassifierFreeGuidance",
+  "_diffusers_version": "0.35.0.dev0",
+  "guidance_rescale": 0.0,
+  "guidance_scale": 10,
+  "start": 0.0,
+  "stop": 1.0,
+  "use_original_formulation": false
+}
+```
+
+#### Switch to a Different Guider Type
+
+Since guiders are `from_config` components (ConfigMixin objects), you can pass guider objects directly to switch between different guidance techniques:
+
+```py
+from diffusers import LayerSkipConfig, PerturbedAttentionGuidance
+config = LayerSkipConfig(indices=[2, 9], fqn="mid_block.attentions.0.transformer_blocks", skip_attention=False, skip_attention_scores=True, skip_ff=False)
+guider = PerturbedAttentionGuidance(
+    guidance_scale=5.0, perturbed_guidance_scale=2.5, perturbed_guidance_config=config
+)
+t2i_pipeline.update_components(guider=guider)
+```
+
+Note that you will get a warning about changing the guider type, which is expected:
+
+```
+ModularPipeline.update_components: adding guider with new type: PerturbedAttentionGuidance, previous type: ClassifierFreeGuidance
+```
+
+<Tip>
+
+💡 **Component Loading Methods**: 
+- For `from_config` components (like guiders, schedulers): You can pass the object directly OR pass a ComponentSpec directly (which calls `create()` under the hood)
+- For `from_pretrained` components (like models): You must use ComponentSpec to ensure proper tagging and loading
+
+</Tip>
+
+Let's verify that the guider has been updated:
+
+```py
+>>> t2i_pipeline.guider
+PerturbedAttentionGuidance {
+  "_class_name": "PerturbedAttentionGuidance",
+  "_diffusers_version": "0.35.0.dev0",
+  "guidance_rescale": 0.0,
+  "guidance_scale": 5.0,
+  "perturbed_guidance_config": {
+    "dropout": 1.0,
+    "fqn": "mid_block.attentions.0.transformer_blocks",
+    "indices": [
+      2,
+      9
+    ],
+    "skip_attention": false,
+    "skip_attention_scores": true,
+    "skip_ff": false
+  },
+  "perturbed_guidance_layers": null,
+  "perturbed_guidance_scale": 2.5,
+  "perturbed_guidance_start": 0.01,
+  "perturbed_guidance_stop": 0.2,
+  "skip_layer_config": [
```

Comment:
@a-r-r-o-w this is duplicated with perturbed_guidance_config and you got a complain when you use this config to create a pag - I think because in the implementation you registered them twice under different names, can you look into this? ( we can fix after the meerge, not super urgent, they just got ignored)